### PR TITLE
Coverity fix in wolfSSL 4.4.0 - part 2

### PR DESCRIPTION
**Pull request by tmael**
This PR fixes Coverity issues in wolfSSL 4.4.0 with the default "./configure --enable-all && make" configurations.

### CODE REVIEW

**Comment by kaleb-himes** [comment]
retest this please.

**Comment by tmael** [comment]
retest this please

File: src/bio.c [hunk 1]
```diff
@@ -165,13 +165,13 @@ static int wolfSSL_BIO_MD_read(WOLFSSL_BIO* bio, void* buf, int sz)
         if (wolfSSL_EVP_DigestSignUpdate((WOLFSSL_EVP_MD_CTX*)bio->ptr, buf,
                         sz) != WOLFSSL_SUCCESS)
         {
-            ret = WOLFSSL_FATAL_ERROR;
+            return WOLFSSL_FATAL_ERROR;
```
Comment (ejohnstown) [inline src/bio.c:168]: As a part of this change, get rid of the ret variable, just use sz.

File: src/ssl.c [hunk 2]
```diff
@@ -23741,7 +23747,7 @@ int wolfSSL_ASN1_TIME_print(WOLFSSL_BIO* bio, const WOLFSSL_ASN1_TIME* asnTime)
     if (wolfSSL_ASN1_TIME_to_string((WOLFSSL_ASN1_TIME*)asnTime, buf,
                 sizeof(buf)) == NULL) {
         XMEMSET(buf, 0, MAX_TIME_STRING_SZ);
-        XMEMCPY(buf, "Bad time value", 14);
+        XMEMCPY(buf, "Bad time value", (int)XSTRLEN("Bad time value"));
```
Comment (ejohnstown) [inline src/ssl.c:23750]: XSTRNCPY(buf, "Bad time value", sizeof(buf)-1);

**Pull request merged**